index.html

```diff
@@ -277,76 +289,94 @@ <h3 class="Skills-title">Skills</h3>
-          to chat please get in touch.
-        </h2>
-        <div class="form">
+      </div>
+    </section>
+
+    <!-- Skills end -->
+
+    <!-- About section end -->
+
+    <!-- Form start  -->
+
+    <form action="https://formspree.io/f/xyyvwazk" method="POST">
+      <h2>
+        I'm always intersted in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <div class="form">
+        <input
+          class="user-name"
+          type="text"
+          name="user_name"
+          placeholder="Full name"
+          maxlength="30"
+          required />
+        <div class="desktop-name">
           <input
+            class="first_name"
             type="text"
-            name="user_name"
-            placeholder="Full name"
+            name="first_name"
+            placeholder="First name"
             maxlength="30"
-            required
-          />
+            required />
           <input
-            type="email"
-            name="email_address"
-            placeholder="Email address"
-            required
-          />
-          <textarea name="message" id="message" maxlength="300" required>Enter text here</textarea>
-          <button type="submit">Get in touch</button>
+            class="last_name"
+            type="text"
+            name="last_name"
+            placeholder="Last name"
+            maxlength="30"
+            required />
         </div>
-      </form>
 
-      <!-- Form end -->
-
-      <!-- Social section start -->
-
-      <section class="social-contact">
-        <ul>
-          <li>
-            <a href="https://twitter.com/Mujeebu93992980">
-              <img src="./images/twitter.svg" alt="" />
-            </a>
-          </li>
-          <li>
-            <a href="https://linkedin.com/in/mujeeb-ur-rahman-54268011a">
-              <img src="./images/Linkedin icon.svg" alt="" />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img src="./images/medium.svg" alt="" />
-            </a>
-          </li>
-          <li>
-            <a href="https://github.com/Mujeeb4582">
-              <img src="./images/github.svg" alt="" />
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <img src="./images/angellist.svg" alt="" />
-            </a>
-          </li>
-        </ul>
-        <div class="email">
-          <a href="mujeeburahman4582@gmail.com">mujeeburahman4582@gmail.com</a>
-        </div>
-      </section>
+        <input
+          type="email"
+          name="email_address"
+          placeholder="Email address"
+          required />
+        <textarea name="message" id="message" maxlength="300" required>
+Typing something</textarea
+        >
+        <button class="dtouch" type="submit">Get in touch</button>
+      </div>
+    </form>
+
+    <!-- Form end -->
+
+    <!-- Social section start -->
+
+    <footer class="social-contact">
```

Comment:
- [ ] There should be a gray line on top of the social media links. Kindly add that gray line to make your portfolio design similar to the Figma template.
 -Your design:
<img width="1391" alt="image" src="https://user-images.githubusercontent.com/61919617/193084460-ec1af1e6-acf9-4901-828e-13d9da817784.png">

-Figma template:
<img width="1112" alt="image" src="https://user-images.githubusercontent.com/61919617/193084517-94346c89-9106-4bb7-8e4b-fd74e56d6a9d.png">